### ISSUE TITLE
Chain Aggro Needs Fixed

### ISSUE THREAD

**Issue by Allelya** [dalaranwow/dalaran-wow#4011]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Mobs from a fair distance away are being aggro'd in succession with the intentional aggro of one mob.

**Expected behaviour**: When a player aggros one mob, another mob that is a fair distance away (more than 10 feet) should NOT be aggro'd as well. With the current stats, it is unnecessarily difficult for players to overcome excessive mobs.

**Steps to reproduce the problem**:

1. Pull a murloc on Darkshore/Pull a Grell in Bashal'aran
2. Watch as another murloc/grell becomes aggro'd even though it's more than 10 feet away
3. Try to overcome the two mobs... die.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
I have no proofs, NPCs or the like for this. Only experience from solo questing.
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
on PTR the settings are:

```sql
#
#    CreatureFamilyAssistanceRadius
#        Description: Distance for creatures calling for assistance from other creatures without
#                     moving.
#        Default:     10 - (Enabled)
#                     0  - (Disabled)

CreatureFamilyAssistanceRadius = 10
```

so this can be changed from server owner only. Thank you for the report, i will be adjusted in the future.
